wm/frame.go

```diff
@@ -388,6 +396,39 @@ func (f *frame) hide() {
 	xproto.UnmapWindow(f.client.wm.x.Conn(), f.client.win)
 }
 
+func (f *frame) changeBorder(){
```

Comment:
This name is a little generic - what does this method do? "applyBorder" or "setupBorder" seems likely candidates.
Also it's a very big method with 2 distinct functions - so probably it should contain just an if that then calls either "setupBorderFramed" and "setupBorderBorderless" or some such thing?
This then suggests an overlap with the decorateBorderless method above, but I didn't fully understand that so can't really say if this looks like duplication or not

Reply:
I changed the name from changeBorder in client.c  to setupBorder.  setupBorder will then determine based on _MOTIF_WM_HINTS whether to addBorder() OR REMOVEbORDER()